src/effects/Paulstretch.cpp

```diff
@@ -167,8 +168,10 @@ bool EffectPaulstretch::Process(EffectInstance &, EffectSettings &)
          const auto pNewTrack = *tempList->Any<WaveTrack>().begin();
          pNewTrack->Flush();
          newT1 = std::max(newT1, mT0 + pNewTrack->GetEndTime());
-         track->Clear(t0, t1);
-         track->Paste(t0, *tempList);
+         PasteTimeWarper warper { t1, t0 + (*tempList->begin())->GetEndTime() };
+         constexpr auto preserve = false;
+         constexpr auto merge = true;
```

Comment:
Why is `preserve` non-default?  Other effects using a time warper pass the default `true` -- except Nyquist, which lets the Lisp code choose.

And now I also see that NoiseReduction uses non-default for `merge` and I question that other inconsistency.

Reply:
Just being extra cautious: the code before didn't preserve splits, and I didn't want to change that behaviour here.

Reply:
Motivation in NoiseReduction.cpp would have been the same, I should think.